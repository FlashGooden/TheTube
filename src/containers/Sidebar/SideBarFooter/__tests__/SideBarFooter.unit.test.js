import React from "react";
import { shallow } from "enzyme";
import { SideBarFooter } from "../SideBarFooter";
import { Menu } from "semantic-ui-react";

describe("SideBarFooter", () => {
  test("should render SidebarFooter", () => {
    const wrapper = shallow(<SideBarFooter />);
    expect(wrapper).toMatchSnapshot();
  });
});
