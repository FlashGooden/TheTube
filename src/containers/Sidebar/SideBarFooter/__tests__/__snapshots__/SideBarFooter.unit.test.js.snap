// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarFooter should render SidebarFooter 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About Press Copywright
    </div>
    <div>
      Creators Advertise
    </div>
    <div>
      Developers +MyTube
    </div>
    <div>
      Legal
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new Features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      All prices include VAT
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Â© Productioncoder.com - A Youtube clone for educational purposes under fair use.
    </div>
  </div>
</Fragment>
`;
